### PR TITLE
Added document for configuration scoping.

### PR DESCRIPTION
**Pull request by jimmyca15**

### CODE REVIEW

**Comment by shirhatti** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[IIS-Administration/api/scope.md](https://github.com/jimmyca15/IIS.Administration-docs/blob/enhance/config_scope/IIS-Administration/api/scope.md) | :white_check_mark:Succeeded | |
[IIS-Administration/api/toc.md](https://github.com/jimmyca15/IIS.Administration-docs/blob/enhance/config_scope/IIS-Administration/api/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C12%5C93859af6-5ab6-bdbe-b7d9-68fc2e7930ed%5CPullRequest%5C201803121639228264-8%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cDtJcbpwGwTrtsykasVAfN5TPPJu0I6M%2BUeJG5%2B16oc%3D&st=2018-03-12T16%3A36%3A05Z&se=2018-04-12T16%3A41%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: IIS-Administration/api/scope.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   * Example: Modifying a setting for *Default Web Site/MyApp/MyFolder* and placing it in the web.config file located in the root of *Default Web Site*
+
+## Targeting a Specific Path Within a Web Application
+
+Querying configuration at a certain path is done by using the *scope* query filter.
+
+Example:
+
+```
+GET https://localhost:55539/api/webserver/http-request-filtering?Default Web Site/MyApp/MyFolder
```
Comment (shirhatti) [inline IIS-Administration/api/scope.md]: Shouldn't this be URL encoded?

Comment (jimmyca15) [reply IIS-Administration/api/scope.md]: Technically yes. I'd like to not URL encode it for the sake of clarity in the example so the path is easily readable.

Comment (shirhatti) [reply IIS-Administration/api/scope.md]: I'd personally still URL encode it or use a website name that doesn't have a space in it.

**Review by shirhatti — APPROVED** [review]

File: IIS-Administration/api/scope.md (hunk 1 above)
Comment (jimmyca15) [reply IIS-Administration/api/scope.md]: And slashes will be %2F%.

Comment (shirhatti) [reply IIS-Administration/api/scope.md]: RIP. Never mind then.

**Comment by shirhatti** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[IIS-Administration/api/scope.md](https://github.com/jimmyca15/IIS.Administration-docs/blob/enhance/config_scope/IIS-Administration/api/scope.md) | :white_check_mark:Succeeded | |
[IIS-Administration/api/toc.md](https://github.com/jimmyca15/IIS.Administration-docs/blob/enhance/config_scope/IIS-Administration/api/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C12%5C93859af6-5ab6-bdbe-b7d9-68fc2e7930ed%5CPullRequest%5C201803121917439234-8%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=byYgZ7y5wj46q5vb8icpAD5vWv%2Fgg94nV56eS5FQboI%3D&st=2018-03-12T19%3A14%3A32Z&se=2018-04-12T19%3A19%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**